packages/pico8-ls/package.yaml

```
@@ -0,0 +1,20 @@
+---
+name: pico8-ls
+description: PICO-8 Language Server
```

I thought the following description was bit more descriptive 🙈:
```suggestion
description: Full language support for the PICO-8 dialect of Lua.
```